### ISSUE TITLE
Setting primaryColor in the config doesn't seem to do anything

### ISSUE THREAD

**Issue by cerino-ligutom** [mermaid-js/mermaid#3242]
**Describe the bug**
I want to apply a theme on my charts globally based on the primary color. From what I understood in the [docs](https://mermaid-js.github.io/mermaid/#/theming?id=theme-variables-reference-table), just setting the `primaryColor` should do the trick and everything else will be derived/calculated. However setting it doesn't do anything.

**To Reproduce**
Try it on the mermaid live editor (v9.1.3 as of reporting):
![image](https://user-images.githubusercontent.com/6721822/179700066-b1700705-e693-481f-86cb-794ad698be4c.png)

**Expected behavior**
Colors would change.

**Code Sample**
[Live editor](https://mermaid.live/edit#pako:eNpVkE1rg0AQhv_KMr2koMf24KHQaJpLIIWGXlwPEx2zS90PxpUi6n_vWimkcxrmfd75mqB2DUEGN0avxKWQVsR4LXPFug8G-0qk6ct8pCCMszTOYr87OtEr5722t8eN36-QyKfTipEIStuvZZPyX__Z0iyK8oQ-OF_dK5dvN4tDqd9VbP9fUUzR9Va2mLWY1sgiR64gAUNsUDdx7Wk1SAiKDEnIYtpQi0MXJCR30ieyxmtH_cpM2xAJnrVBHnPXOd7MD09XfMZWworEA5Y4bPANBjo0OjiGuEjXUwI4BPcx2hqywAP9QYXG-EezFZcf1S9weQ)

**Desktop (please complete the following information):**
 - OS: Windows 10 Home Single Language
 - Browser: Chrome
 - Version: 21H2

**Additional context**
Am I using this wrong? Other theme variables such as `fontFamily` and `fontSize` seem to work but `primaryColor` does nothing.

**Comment by IloBe** [comment]
Hi all, 
Playing around with ER diagrams in gitlab readme markdown and having issues by background color settings of single, specific entities, I found this issue record.

So, first great to find the live editor hint :-) 
There, the primaryColor works by using the base theme and not with other themes; 
evaluating this insight to my readme page:
the same happens by using the %%{init } styling option for the erDiagram inside my markdown file.

For me, it seems to be a 'feature' ;-)
In other words, customization is only possible with base theme.

**Comment by cerino-ligutom** [comment]
> Hi all, Playing around with ER diagrams in gitlab readme markdown and having issues by background color settings of single, specific entities, I found this issue record.
> 
> So, first great to find the live editor hint :-) There, the primaryColor works by using the base theme and not with other themes; evaluating this insight to my readme page: the same happens by using the %%{init } styling option for the erDiagram inside my markdown file.
> 
> For me, it seems to be a 'feature' ;-) In other words, customization is only possible with base theme.

Oh, after setting the `theme` to `base`, it works now! Thank you!
